proto/data.proto

```diff
@@ -145,6 +145,11 @@ message InternalCommitTrigger {
   optional SplitTrigger split_trigger = 1;
   optional MergeTrigger merge_trigger = 2;
   optional ChangeReplicasTrigger change_replicas_trigger = 3;
+
+  // List of intents to resolve on commit or abort. Note that keys
+  // listed here will only be resolved if they fall on the same range
+  // that the transaction was started on.
```

Comment:
If the caller requests intents outside of this range, do we report an error or are they silently ignored?